cli/app.go

```diff
@@ -139,3 +157,37 @@ func runApps(args *docopt.Args, client *controller.Client) error {
 	}
 	return nil
 }
+
+func runInfo(_ *docopt.Args, client *controller.Client) error {
+	appName := mustApp()
+
+	fmt.Println("===", appName)
+
+	w := tabWriter()
+	defer w.Flush()
+
+	if remotes, err := gitRemotes(); err == nil {
+		for _, v := range remotes {
+			if v.Name == appName {
+				listRec(w, "Git Repo:", gitURL(v.Cluster, v.Name))
```

Comment:
> Since this is a URL, call it Git URL (a "git repo" to me is a directory containing .git)

used the [terminology from heroku toolbelt](https://devcenter.heroku.com/articles/using-the-cli#app-commands). Wanna me to change it?